device/common/include/traccc/finding/device/build_tracks.hpp

```diff
@@ -39,9 +39,9 @@ TRACCC_DEVICE inline void build_tracks(
         tips_view,
     track_candidate_container_types::view track_candidates_view,
     vecmem::data::vector_view<unsigned int> valid_indices_view,
-    unsigned int& n_valid_tracks);
+    unsigned int* n_valid_tracks);
```

Comment:
Why? We should only pass variables by pointer that are allowed to be `nullptr`. When something can only be a valid reference, we might as well make it clear in the API. 🤔

Reply:
Fair, updated. :+1: